tests/extensions-are-available/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+    postgres:
+        build:
+          context: ../../PostgresExtended/
+          dockerfile: Postgres-latest.Dockerfile
```

Comment:
Do you think it makes sense to test both versions?

Reply:
Separate ticket - https://github.com/paunin/PostDock/issues/93